archetypes/Exchange/index.tsx

```diff
@@ -23,18 +23,19 @@ const TRADE_OPTIONS = [
     },
 ];
 
-export default (({ onClose }) => {
+export default styled((({ onClose, className }) => {
     const { swapState = swapDefaults, swapDispatch = noDispatch } = useContext(SwapContext);
     const pool = usePool(swapState.selectedPool);
     const receiveIn = useExpectedCommitExecution(pool.lastUpdate, pool.updateInterval, pool.frontRunningInterval);
 
     const amountBN = useBigNumber(swapState.amount);
 
     return (
-        <div className="w-full justify-center sm:mt-14">
-            <Close onClick={onClose} className="absolute right-4 top-4 sm:right-10 sm:top-10 w-3 h-3 cursor-pointer" />
+        <div className={className}>
+            <Close onClick={onClose} className="close" />
+            <h2 className="title">New Commit</h2>
 
-            <div className="flex">
+            <div className="header-wrapper">
```

Comment:
I think its standard to avoid using classNames and just create a styled.div for things like this as well. An [example](https://github.com/Uniswap/interface/blob/main/src/components/CurrencyInputPanel/index.tsx)

Reply:
Cool, I like that. Good design pattern to implement.